@hubblecommerce/hubble/core/pages/_.vue

```diff
@@ -60,6 +60,7 @@ export default {
                         configurator: response.data.configurator,
                         breadcrumb: response.data.breadcrumb
                     });
+                    if (response.data.cmsPage != null) Object.assign(currentPageData, { cmsPage: response.data.cmsPage });
```

Comment:
Assign property directly in object above, inline condition

Reply:
Fixed in [1d37314](https://github.com/hubblecommerce/hubble-frontend-pwa/commit/1d37314b9fdeea90f2a028c4e6ee3d1deaf9c799)